modules/microshift-ingress-controller-config.adoc

```diff
@@ -0,0 +1,150 @@
+====
+
+* The `tuningOptions.headerBufferMaxRewriteBytes` parameter specifies how much memory should be reserved, in bytes, from `headerBufferBytes` for HTTP header rewriting and appending for Ingress Controller connection sessions. The minimum value for `headerBufferMaxRewriteBytes` is `4096`. The `headerBufferBytes` value must be greater than the `headerBufferMaxRewriteBytes` value for incoming HTTP requests.
+* If not set, the default value is `8192` bytes.
++
+[IMPORTANT]
+====
+Setting this field is not recommended because `headerBufferMaxRewriteBytes` parameter values that are too small can break the ingress controller. Conversely, values for `headerBufferMaxRewriteBytes` that are too large could cause the ingress controller to use significantly more memory than necessary.
+====
+
+* The `tuningOptions.healthCheckInterval` parameter specifies how long the router waits between health checks. The default is `5s`.
+
+* The `tuningOptions.serverFinTimeout` parameter specifies how long a connection is held open while waiting for the server response to the client that is closing the connection. The default timeout is `1s`.
+
+* The `tuningOptions.serverTimeout` parameter specifies how long a connection is held open while waiting for a server response. The default timeout is `30s`.
+
+* The `tuningOptions.threadCount` parameter specifies the number of threads to create per HAProxy process. Creating more threads allows each ingress controller pod to handle more connections, at the cost of more system resources being used. `HAProxy` supports up to `64` threads. If this field is empty, default value is `4` threads.
++
+[IMPORTANT]
+====
+Setting this field is not recommended because increasing the number of `HAProxy` threads allows ingress controller pods to use more CPU time under load, and prevent other pods from receiving the CPU resources they need to perform.
+====
+
+* The `tuningOptions.tlsInspectDelay` parameter specifies how long the router can hold data to find a matching route. Setting this value too short can cause the router to fall back to the default certificate for edge-terminated, re-encrypted, or passthrough routes, even when using a better-matched certificate. The default inspect delay is `5s`.
+
+* The `tuningOptions.tunnelTimeout` parameter specifies how long a tunnel connection, including websockets, remains open while the tunnel is idle. The default timeout is `1h`.
+|===
+
+. Complete any other configurations you require, then start or restart {microshift-short} by running one the following commands:
++
+[source,terminal]
+----
+$ sudo systemctl start microshift
+----
++
+[source,terminal]
+----
+$ sudo systemctl restart microshift
+----
+//Q: Anything else we need to do? e.g., other settings we need to make on the network side?
+//Q: Any verification steps we should make before we restart MicroShift?
```

Comment:
after changing ingress configuration  and perform service restart ,
the ingress pod should be restarted with the changes in place,
so we can check if the pod is restarted, using the command:
```bash
> oc get pods -n openshift-ingress
NAME                              READY   STATUS    RESTARTS   AGE
router-default-8649b5bf65-w29cn   1/1     Running   0          6m10s
```
one should look at the AGE field which should be recent